### PR TITLE
Added Anurag Bharadwaj

### PR DESCRIPTION
**Pull request by chatrasen**

### CODE REVIEW

File: README.md~ [hunk 1]
```diff
@@ -0,0 +1,24 @@
+This is a place to practice git. Your first exercise is to add your name to this list.
```
Comment (hargup) [inline README.md~]: This is a probably a backup file created by your editor, you need to remove it.

**Comment by kumarkrishna** [comment]
There are few standard facts that you should follow before commiting and creating a PR:

1. Ensure that the changes you made are actually there. You can verify this by doing a `git diff`.
2. Ensure that no extraneous files are added. This can be avoided by doing `git add -u` (which adds only those files which were already know), and then adding any newer files one by one using `git add`.
3. If you have made several changes across many files but want to commit only one, try using `git add -p`.
4. Lastly, verify on GitHub whether all the changes are pushed.

**Comment by hargup** [comment]
@chatrasen 

Other than what Kumar said,
you also have to make your changes in new branch and you need to keep your master branch clean. When I say clean I mean that you should not make **any** changes in the `master` branch so that for every new pull request you can create a branch out of the master

Since you have made some changes in the `master` you can follow [this](http://stackoverflow.com/a/5916387) answer on Stack Overflow to clean things up.

**Comment by chatrasen** [comment]
i have to keep intact both the master branches of my repo and of AGV?

**Comment by hargup** [comment]
> i have to keep intact both the master branches of my repo and of AGV?

You are not allowed to push to AGV's remote, so it is the master branch of your remote and your local git repository.

**Comment by chatrasen** [comment]
I have again sent the pull request.. It ha not been merged yet

**Pull request closed without merging**